### ISSUE TITLE
Add coordattention in MobileNetV2?

### ISSUE THREAD

**Issue by cc19860606** [Andrew-Qibin/CoordAttention#18]
class InvertedResidual(nn.Module):
    def __init__(self, inp, oup, stride, expand_ratio):
        super(InvertedResidual, self).__init__()
        self.stride = stride
        assert stride in [1, 2]

        hidden_dim = int(round(inp * expand_ratio))
        self.use_res_connect = self.stride == 1 and inp == oup

        layers = []
        if expand_ratio != 1:
            # pw
            layers.append(ConvBNReLU(inp, hidden_dim, kernel_size=1))
        layers.extend([
            # dw
            ConvBNReLU(hidden_dim, hidden_dim, stride=stride, groups=hidden_dim),
            **#AT
            CoordAtt(inp=hidden_dim, oup=hidden_dim),**
            # pw-linear
            nn.Conv2d(hidden_dim, oup, 1, 1, 0, bias=False),            
            nn.BatchNorm2d(oup),
        ])
        self.conv = nn.Sequential(*layers)

    def forward(self, x):
        if self.use_res_connect:
            return x + self.conv(x)
        else:
            return self.conv(x)

**Comment by cc19860606** [comment]
Anyone reproduct the performance of CoordAttention with MobileNetV2 on VOC2007?

**Comment by Andrew-Qibin** [comment]
Have you pretrained the model on ImageNet?

**Comment by cc19860606** [comment]
I firstly test MobileNetV2,the mAP can archive 71.7. Then, I try to add CoordAttention as shown above, meanwhile the pretrained model of 'mobilenet_v2-b0353104.pth' is not use. The mAP only can achive 64. 
@Andrew-Qibin

**Comment by cc19860606** [comment]
ssdlite-pytorch-mobilenext with CoordAttention need to be trained with ImageNet, then test on VOC?

**Comment by Andrew-Qibin** [comment]
Exactly. Otherwise, it does not work. Recently, I will release the pretrained models.

**Comment by cc19860606** [comment]
I look forward to it, thx.

**Comment by cc19860606** [comment]
The ImageNet dataset used for training is ILSVRC2012 or others? And the hyperparameter and code of data augmentation for training?
@Andrew-Qibin
